src/Utils/CameraUtils.js

```diff
@@ -101,13 +100,6 @@ class CameraRig extends THREE.Object3D {
         } else {
             this.camera.matrixWorld.decompose(camera.position, camera.quaternion, camera.scale);
         }
-        view.dispatchEvent({
```

Comment:
this is removed because it's dispatch in the controls (Planar/GlobeControls) directly

Reply:
Oh... You're right, but this dispatch is trigger too often (like I said, when tiles are loading or event when there is no movement). I'll see how I can do